media/css/firefox/mobile.scss

```diff
@@ -196,6 +231,7 @@ html[dir="rtl"] .main-header {
         .header-product-image {
             display: block;
             float: right;
+            margin-top: (($header-product-overflow + $header-product-tablet-padding) * -1);
```

Comment:
I'm wondering if this should be `0` at `$mq-desktop` (and then back to this calculated value at `$mq-desktop-wide`). Currently looks a bit odd/imbalanced at around 1000px wide.

(Not a blocker, and I wouldn't recommend doing a full creative review here. Skip if it's re-engaging a long process.)

![screen shot 2018-02-07 at 1 25 42 pm](https://user-images.githubusercontent.com/317532/35936969-71f87084-0c0a-11e8-97e0-db5719039af6.png)

Reply:
This is updated but not reflected in the diff at this point.